instrumentation/src/test/java/io/opentelemetry/android/OpenTelemetryRumBuilderTest.java

```diff
@@ -132,4 +139,21 @@ void canSetPropagator() {
         TextMapPropagator result = rum.getOpenTelemetry().getPropagators().getTextMapPropagator();
         assertThat(result).isSameAs(customPropagator);
     }
+
+    @Test
+    void setSpanExporterCustomizer() {
+        SpanExporter exporter = mock(SpanExporter.class);
+        Function<SpanExporter, SpanExporter> customizer = x -> exporter;
+        OpenTelemetryRum rum =
+                OpenTelemetryRum.builder(application).addSpanExporterCustomizer(customizer).build();
+        Span span = rum.getOpenTelemetry().getTracer("test").spanBuilder("foo").startSpan();
+        try (Scope scope = span.makeCurrent()) {
+            // no-op
+        } finally {
+            span.end();
+        }
+        // 5 sec is default
+        await().atMost(Duration.ofSeconds(30))
```

Comment:
If I understood correctly, this test would take at least 5 sec to complete? Because if that's the case I think we should find a way to use a simple span processor or something that exports the spans right away. Not necessarily has to be done in this PR though, but I'd like to consider it for a future one.